packages/dds/tree/src/test/util/nestedMap.spec.ts

```diff
@@ -201,4 +202,65 @@ describe("NestedMap unit tests", () => {
 		map.clear();
 		assert.equal(map.size, 0);
 	});
+
+	describe("populateNestedMap", () => {
+		it("can populate an empty nested map", () => {
+			const sourceMap: NestedMap<string, string, number> = new Map<
+				string,
+				Map<string, number>
+			>();
+			setInNestedMap(sourceMap, "Foo", "Bar", 1);
+			setInNestedMap(sourceMap, "Foo", "Baz", 2);
+
+			const destinationMap: NestedMap<string, string, number> = new Map<
+				string,
+				Map<string, number>
+			>();
+
+			populateNestedMap(sourceMap, destinationMap);
+
+			assert.equal(destinationMap.get("Foo")?.get("Bar"), 1);
+			assert.equal(destinationMap.get("Foo")?.get("Baz"), 2);
```

Comment:
I think it would be better to use deepEquals on manually constructed map instead. The code will be cleared and it will also guard against the presence of undesirable values.